<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicineEditView</class>
 <widget class="QWidget" name="MedicineEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>药品信息编辑</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="margin">
    <number>20</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>药品基本信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="margin">
       <number>15</number>
      </property>
      <!-- ID行 -->
      <item row="0" column="0">
       <widget class="QLabel" name="labelID">
        <property name="text">
         <string>ID：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="medEditID">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <!-- 药品名称行 -->
      <item row="1" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>药品名称：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="medEditName">
        <property name="placeholderText">
         <string>请输入药品名称（必填）</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <!-- 类别行 -->
      <item row="2" column="0">
       <widget class="QLabel" name="labelCategory">
        <property name="text">
         <string>类别：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="medComboCategory">
        <item>
         <property name="text">
          <string>抗生素</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>镇痛药</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>感冒药</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>其他</string>
         </property>
        </item>
       </widget>
      </item>
      <!-- 单位行 -->
      <item row="3" column="0">
       <widget class="QLabel" name="labelUnit">
        <property name="text">
         <string>单位：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="medEditUnit">
        <property name="placeholderText">
         <string>如：盒、瓶、片</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <!-- 单价行 -->
      <item row="4" column="0">
       <widget class="QLabel" name="labelPrice">
        <property name="text">
         <string>单价：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="medSpinPrice">
        <property name="minimum">
         <double>0.00</double>
        </property>
        <property name="maximum">
         <double>999999.99</double>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="value">
         <double>0.00</double>
        </property>
       </widget>
      </item>
      <!-- 库存行 -->
      <item row="5" column="0">
       <widget class="QLabel" name="labelStock">
        <property name="text">
         <string>库存：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="medSpinStock">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <!-- 库存状态行（禁用手动选择，由库存数量自动控制） -->
      <item row="6" column="0">
       <widget class="QLabel" name="labelStatus">
        <property name="text">
         <string>库存状态：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="medComboStatus">
        <property name="enabled">
         <bool>false</bool> <!-- 禁用手动选择，由代码自动更新 -->
        </property>
        <item>
         <property name="text">
          <string>正常库存</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>缺货</string>
         </property>
        </item>
       </widget>
      </item>
      <!-- 有效期至行 -->
      <item row="7" column="0">
       <widget class="QLabel" name="labelExpiry">
        <property name="text">
         <string>有效期至：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDateEdit" name="medDateExpiry">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2026</year>
          <month>12</month>
          <day>18</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <!-- 按钮行（简化spacer） -->
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>保存</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
